ui/src/design/Timepicker/Input.tsx

```diff
@@ -95,7 +95,7 @@ const TimePickerInput = React.forwardRef<
         id={id || picker}
         name={name || picker}
         className={twMergeClsx(
-          "focus:bg-accent focus:text-accent-foreground w-[48px] text-center text-base tabular-nums caret-transparent [&::-webkit-inner-spin-button]:appearance-none",
```

Comment:
Since tailwind linting now also works inside`twMergeClsx`, It complained about invalid class names. I removed them as the colors are fine even without defining them.